sdk/containerregistry/azure-containers-containerregistry/src/test/resources/session-records/ContainerRegistryBlobClientIntegrationTests.canUploadHugeBlobInChunksAsync[1].json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Is this using the new recording framework? If not, this isn't proper JSON for the old handling

Reply:
I want to run the test in  live mode only - they test resiliency to huge blobs uopload/download and recordings would be huge. features are covered with unit/smaller tests. 

I have to keep files  around to make integration test infra happy, but they are never used.